torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -343,6 +343,22 @@ inline std::string pickle_str(const c10::IValue& v) {
   return std::string(result.begin(), result.end());
 }
 
+inline std::string get_python_cpp_trace() {
+  std::shared_ptr<torch::CapturedTraceback> tb =
+      torch::CapturedTraceback::gather(
```

Comment:
might be worth warning users, this function can be very slow if getting cpp traces.  (depending on system libs, there are slow/broken versions of addr2line floating around)

Reply:
will add a comment to warn users